File: Ejemplos/UD02/DWES UD02-Ejemplos/DWES UD02-Ej-CS/ud02_01_cs/Codigo fuente auxiliar/Consultas.cs
using System;
using System.Data;
using System.Data.OleDb;

class Consultas : System.Web.UI.Page
{ 
    public static DataTable Vehiculos(string n,string ruta)
    {
        string conexion = "Provider=Microsoft.Jet.OLEDB.4.0; Data Source=" + ruta;
        string sentencia = "select matricula from polizas where nif=@nif";
        OleDbConnection miCnx = new OleDbConnection(conexion);
        OleDbCommand miCmd = new OleDbCommand(sentencia, miCnx);
        miCmd.Parameters.Add("@nif", OleDbType.Char, 9).Value = n;
        DataSet miDSet = new DataSet();
        OleDbDataAdapter Adt = new OleDbDataAdapter(miCmd);
        Adt.Fill(miDSet);
        return miDSet.Tables[0];
    }
}

File: Ejemplos/UD02/DWES UD02-Ejemplos/ud02_01_vb/Codigo fuente auxiliar/Consultas.vb
Imports System
Imports System.Data
Imports System.Data.OleDb

Public Class Consultas
    Inherits System.Web.UI.Page

    Public Shared Function VehiculosCliente(ByVal n As String, ByVal ruta As String) As DataTable
        Dim conexion As String = "Provider=Microsoft.Jet.OLEDB.4.0; Data Source=" & ruta 
        Dim sentencia As String = "select matricula from polizas where nif=@nif"
        Dim miCnx As New OleDbConnection(conexion)
        Dim miCmd As New OleDbCommand(sentencia, miCnx)
        miCmd.Parameters.Add("@nif", OleDbType.Char, 9).Value = n
        Dim miDSet As New DataSet
        Dim Adt As New OleDbDataAdapter(miCmd)
        Adt.Fill(miDSet)
        Return miDSet.Tables(0)
    End Function

End Class
